mlflow/server/js/src/components/CompareRunView.js

```diff
@@ -13,6 +13,9 @@ import { getLatestMetrics } from '../reducers/MetricReducer';
 
 class CompareRunView extends Component {
   static propTypes = {
+    // This prop is optional in case we allow comparing runs across experiments in the future
```

Comment:
As discussed offline. It might be simpler to have this be required for now and then add back the code which allows you to compare across experiments.